src/components/search/findly-sdk.js

```diff
@@ -23728,7 +23707,7 @@ if(res?.graph_answer?.payload?.center_panel){
     let title = res?.graph_answer?.payload?.center_panel?.data[0]?.snippet_title?helpers.convertMDtoHTML(res?.graph_answer?.payload?.center_panel?.data[0]?.snippet_title) : '';
     snippetObj = {'title':title,
     'answer':listSnippetData, page_url:res?.graph_answer?.payload?.center_panel?.data[0]?.url,
-    'source':res?.graph_answer?.payload?.center_panel?.data[0]?.source_title || res?.graph_answer?.payload?.center_panel?.data[0]?.source,
+    'source':undefined, //res?.graph_answer?.payload?.center_panel?.data[0]?.source_title || res?.graph_answer?.payload?.center_panel?.data[0]?.source,
```

Comment:
@VenkatIc-Kore, is there any reason for changing the 'source' property to undefined?
CC: @SunilSi-kore